### PR TITLE
Make sure cron jobs are never returning errors

### PR DESCRIPTION
**Pull request by noboruma**

### CODE REVIEW

File: deepfence_worker/worker.go [hunk 1]
```diff
@@ -285,55 +285,58 @@ func startWorker(wml watermill.LoggerAdapter, cfg config) error {
 	worker := NewWorker(wml, cfg, mux)
 
 	// sbom
-	worker.AddNoPublisherHandler(utils.ScanSBOMTask, sbom.NewSBOMScanner(ingestC).ScanSBOM, false)
+	worker.AddNoPublisherHandler(utils.ScanSBOMTask, LogErrorWrapper(sbom.NewSBOMScanner(ingestC).ScanSBOM), false)
+
 	worker.AddHandler(utils.GenerateSBOMTask, sbom.NewSbomGenerator(ingestC).GenerateSbom,
 		utils.ScanSBOMTask, publisher)
```
Comment (noboruma) [inline deepfence_worker/worker.go:291]: @gnmahanth is this function safe?
I see it returns errors

Comment (gnmahanth) [reply deepfence_worker/worker.go:291]: add wrapper here

**Review by gnmahanth — APPROVED** [review]

**Comment by shyam-dev** [comment]
We should have a way to propagate relevant errors back to the UI/users

**Comment by noboruma** [comment]
Very good point @shyam-dev 
We should have an error page on the console to make it easy to checks errors & report them if needed

**Pull request merged**